addons/api/addon/models/worker.js

```diff
@@ -55,6 +65,34 @@ export default class WorkerModel extends GeneratedWorkerModel {
     );
   }
 
+  /**
+   * Returns the api tags as an array of key/value pair objects.
+   * @type {[object]}
+   */
+  getApiTagList() {
+    if (!this.api_tags) {
+      return null;
+    }
+
+    return Object.entries(this.api_tags).flatMap(([key, value]) =>
+      value.map((tag) => ({ key, value: tag })),
+    );
+  }
+
+  /**
+   * Returns all tags as an array of key/value pair objects with tag type.
+   * @type {[object]}
+   */
+  getAllTags() {
+    return [
+      ...(this.getConfigTagList() || []).map((tag) => ({
+        ...tag,
+        type: 'config',
+      })),
+      ...(this.getApiTagList() || []).map((tag) => ({ ...tag, type: 'api' })),
```

Comment:
Could the type have just been added in the `getApiTagList` and `getConfigTagList` instead of here and then you'd just only merge the two?

Reply:
Originally I didn't want to modify the output of getConfigTagList but just adding `type` doesn't hurt anything. Thanks for the suggestion, I will update now.